documentation/userguide.md

```diff
@@ -244,6 +244,8 @@ Things to check if no trace files are written:
 In case the application doesn't start at all, please check the file `C:\Users\Public\profiler_debug.log`.
 It may contain stack traces in case the profiler crashed.
 
+The `attach.log` file in the directory of the profiler dlls might also provide some insights. It contains information about processes to which the profiler attached to.
```

Comment:
Join this paragraph and the previous, as they are both concerned with debugging startup failure. Please consistently capitalize "DLLs". The last "to" is too much, just say "to which the profiler attached".